### ISSUE TITLE
Create a Doxumentation Team

### ISSUE THREAD

**Issue by gbowne1** [taotnpwaft/docs#11]
The group Documentation needs a small team.

We have 14 members currently.

Would anyone like to help with creating Documentation for the group?

**Comment by hitblast** [comment]
Am in. Also @gbowne1 I would like you to know that there are many Discussions in this organisation which haven't been marked "answered" whereas they do actually have a pretty good conversation in them. It'd be great if some of you took the time to mark them as answered.

**Comment by gbowne1** [comment]
@hitblast 

   Ok, I'll add you to the Documentation team.. but I think you can request to be part of a team in the Teams tab.

I will look into the Discussions thing.  Many of the discussions don't have much conversation yet.  I did not want to clutter the projects with Issues that are actually just discussions or the opposite.

Thanks for the reccomendation.

**Comment by gbowne1** [comment]
@hitblast 

An update...After reading through the GitHub docs on marking answers and unless I am seeing things wrong, I could not find anything of what they are describing is needed for doing that in any of our discussions.

I do not even see the place where it says mark as answered.  

That being said, Issues and Discussions and even Projects are tied together it seems in Org mode.

**Comment by hitblast** [comment]
@gbowne1 Alright then.

**Comment by gbowne1** [comment]
Yes.  I saw that later on in my research, but the GitHub docs on this were a little vague.

**Comment by gbowne1** [comment]
> @gbowne1 Thanks. Oh speaking of marking as answered, those can only be done to a discussion that derives from the **Q&A** category, much like [this one](https://github.com/taotnpwaft/discordbot/discussions/7) right here.

I changed the category of the ones that were mainy questions that I've asked to the Q&A category and have answers.

**Comment by hitblast** [comment]
> > @gbowne1 Thanks. Oh speaking of marking as answered, those can only be done to a discussion that derives from the **Q&A** category, much like [this one](https://github.com/taotnpwaft/discordbot/discussions/7) right here.
> 
> I changed the category of the ones that were mainy questions that I've asked to the Q&A category and have answers.

Looks neat.

**Comment by gbowne1** [comment]
I forgot you could change categories.

My bad.

Yep still learning GitHub.

**Comment by gbowne1** [comment]
@hitblast I changed your status to Owner from Member.

We need to get organizing this mess of an organizaton that I created in late 2019.  It's mostly been me working on this group.

I've been looking to find someone to help with that, and not just working on the bots or the site.

I have some suspicions as to whats happened too.  At least one person thinks its "over engineered".

**Comment by hitblast** [comment]
> @hitblast I changed your status to Owner from Member.
> 
> We need to get organizing this mess of an organizaton that I created in late 2019. It's mostly been me working on this group.
> 
> I've been looking to find someone to help with that, and not just working on the bots or the site.
> 
> I have some suspicions as to whats happened too. At least one person thinks its "over engineered".

Understandable.

**Comment by gbowne1** [comment]
Yeah.  I was about ready to hire on Fiverr ir something, or recruit on a subreddit that would bring us someone to help lead or help organize this.

@swathe removed himself and @idormain asked to be removed.

Several others said they just didnt have time to participate.  Mostly time related.
